habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/PublishToPyPiRepoMojo.java

```diff
@@ -90,6 +102,27 @@ public class PublishToPyPiRepoMojo extends AbstractHabushuMojo {
     @Parameter(property = "habushu.devRepositoryUrlUploadSuffix", defaultValue = "legacy/")
     protected String devRepositoryUrlUploadSuffix;
 
+    /**
+     * Specifies the number of times a push to the configured PyPI repository will be attempted before stopping (inclusive
+     * of the initial attempt). While this defaults to three and is fully configurable, it can be set to zero to never
+     * retry or set to any negative number for unlimited retries.  Unlimited retries will follow a fibonacci backoff
+     * interval.
+     */
+    @Parameter(property = "habushu.pyPiPushRetries", defaultValue = "3")
+    protected int pypiPushRetries = 3;
+
+    /**
+     * A multiplier to apply to the retry wait time (e.g., round of fibonacci).
+     */
+    @Parameter(property = "habushu.pypiPushRetryMultiplier", defaultValue = "15000")
```

Comment:
A: This one is way more obvious based on the default, but again might be worth saying the unit is `ms`